src/domain/Avatars/AvatarEntity/AvatarEntity.tsx

```diff
@@ -131,57 +103,76 @@ export class AvatarEntity extends React.PureComponent<IAvatarEntityProps> {
     }
 
     return (
-      <SecondaryTextWrapper
-        textType={secondaryTextType!}
-        isCompact={isCompact}
-        secondaryColor={secondaryColor!}
-        secondaryWeight={secondaryWeight!}
+      <Text
+        isTruncated
+        color={Variables.Color.n700}
+        margins={isCompact ? { left: Variables.Spacing.sXSmall } : undefined}
+        type={size === Size.Normal ? Props.TypographyType.Small : Props.TypographyType.XSmall}
       >
         {text}
-      </SecondaryTextWrapper>
+      </Text>
     )
   }
 
   get tertiaryText (): JSX.Element | null {
     const {
       tertiaryText,
       isCompact,
-      tertiaryTextType
+      size
     } = this.props
 
     if (!tertiaryText || isCompact) {
       return null
     }
 
     return (
-        <TertiaryTextWrapper
-          textType={tertiaryTextType!}
-          isCompact={isCompact}
-        >
-          {tertiaryText}
-        </TertiaryTextWrapper>
+      <Text
+        isInline={false}
+        isTruncated
+        color={Variables.Color.n700}
+        margins={{ left: size === Size.Normal ? 48 : 38 }}
+        type={size === Size.Normal ? Props.TypographyType.Small : Props.TypographyType.XSmall}
+      >
```

Comment:
I don't think it should be the text component's job to position itself here. 
I would wrap it with a styled div for positioning so you wouldn't need to pass through `isInline` or `margins` here